test/cases/api.sh

```diff
@@ -57,13 +61,21 @@ function checkEnvGCP() {
   printenv GOOGLE_APPLICATION_CREDENTIALS GCP_BUCKET GCP_REGION GCP_API_TEST_SHARE_ACCOUNT > /dev/null
 }
 
+# Check that needed variables are set to access Azure.
+function checkEnvAzure() {
+  printenv AZURE_TENANT_ID AZURE_SUBSCRIPTION_ID AZURE_RESOURCE_GROUP AZURE_LOCATION AZURE_CLIENT_ID AZURE_CLIENT_SECRET > /dev/null
+}
+
 case $CLOUD_PROVIDER in
   "$CLOUD_PROVIDER_AWS")
     checkEnvAWS
     ;;
   "$CLOUD_PROVIDER_GCP")
     checkEnvGCP
     ;;
+  "$CLOUD_PROVIDER_GCP")
```

Comment:
Copy&Paste error, should be `"$CLOUD_PROVIDER_AZURE"`

Reply:
good catch, thanks!